testing/lib.rs

```diff
@@ -58,6 +58,7 @@ integration_test!(
   error_ext_stack,
   error_with_stack,
   error_without_stack,
+  pending_unref_op_tla,
```

Comment:
Could you paste the current panic out that's happening right now? I don't see a scenario that would panic, but rather deadlock.

Reply:
It's actually "Top-level await promise never resolved" -- the test I added was panicking at one point. :)